### PR TITLE
Created Basic Profile Model

### PR DESCRIPTION
**Pull request by rajivtitus**
### What this PR does (required):
- Created initial Profile model

### Screenshots / Videos (required):
- N/A

### Any information needed to test this feature (required):
- Open file `server/models/Profile.js`

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

**Review by dproc96 — COMMENTED** [review]
This looks good so far, I think it would also be good to establish a linkage between the User model and the corresponding Profile. Maybe a reference in the User model or in both models, think through how you would need to query it to decide one way or the other

**Comment by rajivtitus** [comment]
Hello @dproc96, I have added a reference to the profile model from the user model.
Thanks!

File: server/models/Profile.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+    },
+    gender: String,
+    birthDate: Date,
```
Comment (sina-jamshidi) [inline server/models/Profile.js]: You can remove these fields.

**Review by sina-jamshidi — APPROVED** [review]
LGTM

**Review by dproc96 — APPROVED** [review]

**Pull request merged**